src/async_impl/client.rs

```diff
@@ -1406,6 +1432,35 @@ impl ClientBuilder {
         self
     }
 
+    /// Add a certificate revocation list.
+    ///
+    ///
+    /// # Optional
+    ///
+    /// This requires the `rustls-tls(-...)` Cargo feature enabled.
+    #[cfg(feature = "__rustls")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "rustls-tls")))]
+    pub fn add_crl(mut self, crl: CertificateRevocationList) -> ClientBuilder {
+        self.config.crls.push(crl);
+        self
+    }
+
+    /// Add multiple certificate revocation lists.
+    ///
+    ///
+    /// # Optional
+    ///
+    /// This requires the `rustls-tls(-...)` Cargo feature enabled.
+    #[cfg(feature = "__rustls")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "rustls-tls")))]
+    pub fn add_crls(
```

Comment:
I think I'd prefer to leave this method off, to start with. We can always add it later.

Reply:
With `add_crls` the code looks more concise and clear
```
        let crls = CertificateRevocationList::from_pem_bundle(&buf)?;

        let client =  Client::builder()
          .use_rustls_tls()
          .tls_built_in_root_certs(true)
          .add_root_certificate(server_cert)
          .identity(identity)
          .add_crls(crls);
          .build()?;
```

without `add_crls`
```
        let mut builder = Client::builder();

        let crls = CertificateRevocationList::from_pem_bundle(&buf)?;
        for crl in crls {
            builder = builder.add_crl(crl);
        }

        let client = builder
            .use_rustls_tls()
            .tls_built_in_root_certs(true)
            .add_root_certificate(server_cert)
            .identity(identity)
            .build()?;
```

I suggested the method as I was going to use it projects I work on. The decision is up to you.

Reply:
@seanmonstar what is your decision?